server/app/views/components/Icons.java

```diff
@@ -147,6 +147,20 @@ public enum Icons {
           + " 12.583Q7.562 12.583 7.562 12.583Q7.562 12.583 7.562 12.583Q7.562 12.583 7.562"
           + " 12.583Q7.562 12.583 7.562 12.583Q7.562 12.583 7.562 12.583Q7.562 12.583 7.562"
           + " 12.583Q7.562 12.583 7.562 12.583Q7.562 12.583 7.562 12.583Z"),
+  // Help https://fonts.google.com/icons?icon.query=help
```

Comment:
question (non-blocking): There's a couple different icons on this page, which one is it?

Reply:
Added a little more help to the comment.